### DISCUSSION TITLE
Patching Ims for VoLTE

### DISCUSSION THREAD

**Discussion by orkunergun** [ROM-builders/temporary#10570] [Q&A] [answered]
Hello there! I need your help for patching IMS. I can't remove frameworks/opt/net/ims tracked by Aosp. When I try to remove it, it gives the error "fatal: remove-project element specifies non-existent project: android_frameworks_opt_net_ims" . I will be waiting for your help as soon as possible. Thanks in advance!

**Comment by Apon77** [comment, marked as answer]
> Hello there! I need your help for patching IMS. I can't remove frameworks/opt/net/ims tracked by Aosp. When I try to remove it, it gives the error "fatal: remove-project element specifies non-existent project: android_frameworks_opt_net_ims" . I will be waiting for your help as soon as possible. Thanks in advance!

check #tutorial and #local_manifests in telegram group

**Reply by orkunergun** [reply]
Thanks anyways I found solution for this
